### ISSUE TITLE
Browser locks up when closing server that is connected to remotely over TLS

### ISSUE THREAD

**Issue by mofojed** [deephaven/deephaven-core#5413]
**Description**

When using `uri.resolve` to connect to a remote table over TLS, it blows up if the origin server dies.

**Steps to reproduce**
1. In your `deephaven.prop` file (`~/.config/deephaven/deephaven.prop` on Linux), add the following to enable TLS and anonymous authentication (you'll need to update the paths to your deephaven-core repo):
```
http.port=8443
http.http1=false

ssl.identity.type=privatekey
ssl.identity.certChainPath=/path/to/deephaven-core/server/dev-certs/server.chain.crt
ssl.identity.privateKeyPath=/path/to/deephaven-core/server/dev-certs/server.key

outbound.ssl.trust.type=certs
outbound.ssl.trust.path=/path/to/deephaven-core/server/dev-certs/ca.crt

AuthHandlers=io.deephaven.auth.AnonymousAuthenticationHandler
```
2. Start up one deephaven server on 10001: `deephaven server --port 10001`
3. In another terminal, start up a second server on 10002: `deephaven server --port 10002`
4. In the browser, go to https://localhost:10001, and run the following:
```python
from deephaven import time_table

tt = time_table("PT1s").tail(1)
```
5. In another tab, go to https://localhost:10002, and run the following:
```
from deephaven import uri

remote_tt = uri.resolve("dh://localhost:10001/scope/tt")
```
6. Kill the server on port 10001 (the origin server)

**Expected results**
6. Table errors out in browser UI of 10002, but UI remains responsive

**Actual results**
6. Table errors out in browser UI of 10002, but UI completely locks up and browser freezes.

**Additional details and attachments**

If applicable, add any additional screenshots, logs, or other attachments to help explain your problem.

Seems to get stuck in a loop:
![image (14)](https://github.com/deephaven/deephaven-core/assets/4505624/41ab781b-1a8c-49de-8333-1a729c9707e8)

**Versions**
`66cede6cb0bfe651119954aac4b56d92d75724e8`
Engine Version: 0.34.0-SNAPSHOT
Web UI Version: 0.74.0
Java Version: 11.0.22
Barrage Version: 0.6.0
Browser Name: Chrome 124
OS Name: Linux

**Comment by devinrsmith** [comment]
I'm not sure if TLS was ultimately required here or not. Ultimately, this reproducer triggered https://github.com/deephaven/deephaven-core/issues/5417

**Comment by rcaudy** [comment]
We think this was addressed by the solution to #5417.
